solidity/dashboard/src/pages/tbtc-migration/TokenUpgradePortalPage.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from "react"
+import {
+  MigrationPortalForm,
+  ConfirmMigrationModal,
+} from "../../components/tbtc-migration"
+import { TBTC } from "../../utils/token.utils"
+import { useModal } from "../../hooks/useModal"
+import { useWeb3Address } from "../../components/WithWeb3Context"
+import { useDispatch, useSelector } from "react-redux"
+import { tbtcV2Migration } from "../../actions"
+
+const TokenUpgradePortalPage = () => {
+  const { openConfirmationModal } = useModal()
+  const address = useWeb3Address()
+  const dispatch = useDispatch()
+
+  const { tbtcV1Balance, tbtcV2Balance, unmintFee } = useSelector(
+    (state) => state.tbtcV2Migration
+  )
+
+  useEffect(() => {
+    dispatch(tbtcV2Migration.fetchDataRequest(address))
+  }, [address, dispatch])
+
+  const onSubmitMigrationForm = async (values, awaitingPromise) => {
+    const { amount, from, to } = values
+    const _amount = TBTC.fromTokenUnit(amount).toString()
+
+    await openConfirmationModal(
+      {
+        modalOptions: { title: to === "v2" ? "Upgrade" : "Downgrade" },
+        from,
+        to,
+        amount: _amount,
+        fee: unmintFee,
+      },
+      ConfirmMigrationModal
+    )
```

Comment:
This modal seems too wide when comparing to the Figma views. I think we should set `480px` width here.

One way to do it would be to set the className in modalOptions:
```
modalOptions: {
   title: to === "v2" ? "Upgrade" : "Downgrade"
   classes: {
      modalWrapperClassName: "modal-wrapper__confirm-migration-modal",
   },
}
```

and then set the width in .less file